src/Controls/Maps/src/MapElement.cs

```diff
@@ -7,6 +7,9 @@
 
 namespace Microsoft.Maui.Controls.Maps
 {
+	/// <summary>
+	/// Represents an element which is visually drawn on the map control.
```

Comment:
Maybe add a cref to the word map?
```suggestion
	/// Represents an element which is visually drawn on the <see cref="Map"/> control.
```